content/chainguard/chainguard-images/registry/authenticating.md

```diff
@@ -21,6 +21,12 @@ However, logging in with a Chainguard account and authenticating when pulling fr
 
 This may enable Chainguard to notify you of upcoming deprecations, changes in behavior, critical vulnerabilities and remediations for images you have recently pulled.
 
+## Signing Up
+
+You can register a Chainguard account by signing up through our [sign up form](https://console.enforce.dev/auth/login?utm_source=docs). This will create your account and your organization's root [IAM group](/chainguard/chainguard-enforce/iam-groups/overview-of-enforce-iam-model/). If you already have an account, you can log in through the [login page]((https://console.enforce.dev/auth/login?utm_source=docs).
```

Comment:
```suggestion
You can register a Chainguard account through our [sign up form](https://console.enforce.dev/auth/login?utm_source=docs). This will create your account and your organization's root [IAM group](/chainguard/chainguard-enforce/iam-groups/overview-of-enforce-iam-model/). If you already have an account, you can log in through the [login page]((https://console.enforce.dev/auth/login?utm_source=docs).
```

"signing up through our sign up form" seems repetitive